### PR TITLE
Pws white space fix

### PR DESCRIPTION
**Pull request by emreyigit**
I faced a problem when a java class path contains white spaces in it, controller and server doesn't work. Quoted the all usages of `$script:options.classpath`, and tested with run-server and test commands.

### CODE REVIEW

File: hz.ps1 [hunk 1]
```diff
@@ -555,6 +555,9 @@ function ensure-jar ( $jar, $repo, $artifact ) {
     $classpath = $script:options.classpath
     if (-not [System.String]::IsNullOrWhiteSpace($classpath)) { $classpath += $s }
     $classpath += "$tmpDir/lib/$jar"
+    # condiser to quote the path to escape from white space
```
Comment (zpqrtbnk) [inline hz.ps1:558]: You probably want this to be "consider" ;) And, maybe it's even "be sure to..."?

**Comment by zpqrtbnk** [comment]
Have added 1 minor comment for you, rest sounds good.

File: hz.ps1 (hunk 1 above)
Comment (emreyigit) [reply hz.ps1:558]: You are right, that could better :)

**Review by zpqrtbnk — APPROVED** [review]
Looks good to me.

**Pull request merged**